client-sqldelight-gradle-plugin/src/test/projects/happyPath/lib/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import app.cash.backfila.client.sqldelight.plugin.SqlDelightRecordSource
+
+plugins {
+  id("app.cash.backfila.client.sqldelight")
+  id("app.cash.sqldelight")
+  kotlin("jvm")
+}
+
+sqldelight {
+  databases {
+    create("HockeyDataDatabase") {
+      packageName.set("app.cash.backfila.client.sqldelight.hockeydata")
+      dialect(libs.sqldelightMysqlDialect)
+      srcDirs.setFrom(listOf("resources/migrations"))
+      deriveSchemaFromMigrations.set(true)
+      migrationOutputDirectory.set(file("$buildDir/resources/main/migrations"))
+      verifyMigrations.set(true)
+    }
+  }
+}
+
+backfilaSqlDelight {
+  addRecordSource(
+    name = "hockeyPlayerOrigin",
+    database = "app.cash.backfila.client.sqldelight.hockeydata.HockeyDataDatabase",
+    tableName = "hockeyPlayer",
+    keyName = "player_number",
```

Comment:
Should it be `keyColumnName` since this is the column name?

Reply:
It could eventually be compound or some other kind of key. I'm not sure I want to change this.